Formula/purple-discord.rb

```diff
@@ -0,0 +1,20 @@
+class PurpleDiscord < Formula
+  desc "Discord support for Pidgin"
+  homepage "https://github.com/EionRobb/purple-discord"
+  head "https://github.com/EionRobb/purple-discord.git"
```

Comment:
I'm afraid we won't accept head-only formulae in Homebrew/core.

Reply:
So the only way is with a separate tap?  Surely there's other projects/plugins out there that aren't cutting release tarballs?

Reply:
> Surely there's other projects/plugins out there that aren't cutting release tarballs?

Yup but we don't accept those projects' inclusion into Homebrew/core. If they are HEAD there's no difference to being in a tap; you aren't getting a binary package anyway (and it means we don't get issues if upstream breaks compilation).

Reply:
An option could be pinning to a specific `revision` and making them all `resource`s inside the main `pidgin` formula (so it comes with these by default).

Reply:
So to be clear.  If these packages had tags that we could pull tarballs for, would that be enough for inclusion?

Reply:
Uh.. that's not how we do things.  Also quite of few of these are the result of reverse engineering network protocols and as such, can change at any time, and need to be out of tree so that they can be quickly fixed and not force an update to the core application that might not be ready for release.

In other words, modular components are better in this case.

Reply:
Fine with me. My suggestion on "batteries included" is based on how we (Homebrew) do things and what will result in a better experience for our users. I am happy to compromise if these formulae can have stable release tarballs made.